ios/NativeSigner/AppDelegate.m

```diff
@@ -34,7 +34,7 @@ - (BOOL)application:(UIApplication *)application didFinishLaunchingWithOptions:(
 - (NSURL *)sourceURLForBridge:(RCTBridge *)bridge
 {
 #if DEBUG
-  return [[RCTBundleURLProvider sharedSettings] jsBundleURLForBundleRoot:@"index" fallbackResource:nil];
+  return [NSURL URLWithString:@"http://192.168.1.220:8081/index.ios.bundle?platform=ios&dev=true"];
```

Comment:
I'm not able to get my metro bundler to connect and run, and I suspect this is the issue...Ran the build-prod:ios script after a cleaning cache as well. @hanwencheng

Reply:
just did a clean build reverting this line and it works :)

Reply:
My fault, this line cause problem, and should not be here.